thirdparty/runtime_environments.go

```diff
@@ -14,18 +14,27 @@ import (
 	"github.com/pkg/errors"
 )
 
+type ImageEventEntryAction string
+
+const (
+	ImageEventEntryActionAdded   ImageEventEntryAction = "ADDED"
+	ImageEventEntryActionUpdated ImageEventEntryAction = "UPDATED"
+	ImageEventEntryActionDeleted ImageEventEntryAction = "DELETED"
+)
+
+type ImageEventType string
```

Comment:
Is there a reason we want to make this and not just use string?

Reply:
I did this so that I could map the ImageEventType to the graphql enum ImageEventType in graphql/schema/types/image.graphql.